src/frontend/Komponenter/Behandling/VedtakOgBeregning/Barnetilsyn/VedtaksoppsummeringBarnetilsyn.tsx

```diff
@@ -65,9 +91,22 @@ export const VedtaksoppsummeringBarnetilsyn: React.FC<{
             </OppsummeringContainer>
             {finnesBarnPåBehandling && (
                 <OppsummeringContainer>
-                    {barnPåBehandling.map((barn, index) => (
-                        <OppsummeringAvBarn barn={barn} key={barn.personIdent + `${index}`} />
-                    ))}
+                    {barnPåBehandling.map((barn) => {
+                        const oppfyllerKritereForAleneomsorg =
+                            vilkårsresultatAleneomsorgPerBarn[barn.barnId] ===
+                            Vilkårsresultat.OPPFYLT;
+                        const oppfyllerKriterieForAlderPåBarn =
+                            vilkårsresultatAlderPåBarnPerBarn[barn.barnId] ===
+                            Vilkårsresultat.OPPFYLT;
+                        return (
+                            <OppsummeringAvBarn
+                                key={barn.barnId}
+                                barn={barn}
+                                oppfyllerKriterieForAleneomsorg={oppfyllerKritereForAleneomsorg}
+                                oppfyllerKriterieForAlderPåBarn={oppfyllerKriterieForAlderPåBarn}
+                            />
```

Comment:
istedenfor "oppfyllerKriterieForAleneomsorg" så kan vi kanskje bare sende inn vilkårsresultat direkte til Oppsummering av barn? We can fix